shared/utility/support/evil/pure_evil.hpp

```diff
@@ -25,28 +23,28 @@
 #include <vector>
 
 // If we are not in debug mode, don't build it! Please don't build it!
-#ifndef NDEBUG
+#if !defined(NDEBUG) and !defined(__APPLE__)
 
 namespace utility::support::evil {
 
     struct StackFrame {
 
         StackFrame() = default;
 
-        StackFrame(uintptr_t pc_, std::string file_, int lineno_, std::string function_)
-            : pc(pc_), file(std::move(std::move(file_))), lineno(lineno_), function(std::move(function_)) {}
+        StackFrame(uintptr_t pc, std::string file, int lineno, std::string function)
+            : pc(pc), file(std::move(file)), lineno(lineno), function(std::move(function)) {}
```

Comment:
@KipHamiltons Didn't you say this was bad?

Reply:
Yes, it is bad. Not sure how this got through lol

Reply:
For the record, this doesn't shadow variables since in context the meaning of both is entirely defined. Up to you if you want to add the underscores to the end. I don't generally bother unless I have a specific need to refer to both individually within the constructor body itself